data-collection/deploy/module-aws-feeds.yaml

```diff
@@ -136,81 +147,85 @@ Resources:
               return parser.text.strip() + '\n\n' + '\n'.join([f"[{index}]: {url}" for index, url in parser.ref.items()])
 
           def lambda_handler(event, context):
-              feed_url = os.environ['FEED_URL']
+              list = os.environ['FEEDS_LIST'].split(',')
```

Comment:
maybe call it feed_list to avoid naming confusion with Python list

Reply:
+1 here